client/my-sites/backup/rewind-flow/rewind-config-editor.tsx

```diff
@@ -26,23 +26,15 @@ const BackupRewindConfigEditor: FunctionComponent< Props > = ( {
 	const checkboxes = [
 		{
 			name: 'themes',
-			label: translate( '{{strong}}WordPress Themes{{/strong}}', {
+			label: translate( '{{strong}}WordPress themes{{/strong}}', {
 				components: {
 					strong: <strong />,
 				},
 			} ),
 		},
 		{
 			name: 'plugins',
-			label: translate( '{{strong}}WordPress Plugins{{/strong}}', {
-				components: {
-					strong: <strong />,
-				},
-			} ),
-		},
-		{
-			name: 'uploads',
-			label: translate( '{{strong}}Media Uploads{{/strong}}', {
+			label: translate( '{{strong}}WordPress plugins{{/strong}}', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( '{{strong}}WordPress Plugins{{/strong}}' )` *ES Score: 9*